public/locales/en/translation.json

```diff
@@ -6,5 +6,62 @@
 			"loans": "LOANS",
 			"support": "SUPPORT"
 		}
+	},
```

Comment:
I wouldn't put the translation in uppercase. I think this can be done in CSS instead.

Reply:
will fix this in a new PR.